dashboard/src/utils/addDay.js

```diff
@@ -0,0 +1,9 @@
+const addOneDay = (date) => {
+  const newDate = new Date(date.setDate(date.getDate() + 1));
```

Comment:
This function modifies its input, bumping it forward by one day, and then creates a new (temporary) `Date` object which is formats into a string.  This is likely to be a surprise to the calling code.

I assume that this function should make a copy of `date` first (i.e., by using it to instantiate a new `Date` object...) and then modify the _copy_ (otherwise, we don't need the copy!).

Reply:
Also, I think we should be using `getUTCDate()` and `setUTCDate()` here, for consistency if for no other reason.